clientd3d/dialog.c

```diff
@@ -663,6 +674,43 @@ void DisplayDescription(object_node *obj, BYTE flags, char *description,
 }
 /************************************************************************/
 /*
+* GetRaritySuffix:  Provide a rarity suffix based on the given rarity type.
+*/
+std::string GetRaritySuffix(int rarity)
+{
+	std::string suffix;
```

Comment:
You don't really need a variable here.

Reply:
Removed the suffix variable and changed the return to a GetString call.